.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+contact_links:
+    - name: 📃 Blank Issue
+      url: https://github.com/PalisadoesFoundation/talawa-admin/issues/new
```

Comment:
No blank issue creation. We need a version of the existing template text that will cover both bugs and features.
The use of the guiding text has been very useful in creating issues and PRs that are easier to understand due to the better documentation

Reply:
For the pre-populated text, we currently have bug and feature, how do I implement them in one? or do i make 2 blank

Reply:
Added existing and new templates - [check here](https://github.com/pranshugupta54/talawa-admin/issues/new/choose)

<img width="1333" alt="image" src="https://github.com/PalisadoesFoundation/talawa-admin/assets/76090263/54aa7bf7-dc8b-47b1-95ea-9f349fd901f9">